Create Schema on Strapi

imApoorva36 opened the pull request:
We have added entire schema as per the [db diagram](https://dbdiagram.io/d/Web-Club-Website-Schema-65b7eacbac844320aef8b83b) , other than the roles table and it's relation which will be done after it's finalised .

Please review and let us know if there are any other corrections.

Commit:
collections_added

imApoorva36 committed:
added schema other than roles

j-hariharan commented:
The following relations have wrong cardinalities assigned to then (many-to-one, one-to-one, etc.,.). So these relationships need fixing:
1. member - sig
2. member - blog
3. sig - blog
4. sig - event

Some other minor changes to be made include:
1. members field rename to "written_by" in blog
2. written_on and written_by fields in reading_list (if necessary)
3. WECConfiguration schema in strapi is taken as a "single type". This would work if each config variable was a separate field, but with the current schema, it needs to be added as a "collection type" itself.
4. not sure if there is need for SIG field in achievements
5. event_type might be better as enum type in event table
6. should resources field in events table be an array?
7. ~look into rich text~ can be done later

Commit:
strapi_relations_updated

j-hariharan committed:
Edited schema according to PR comment
(https://github.com/WebClub-NITK/wec-nitk-website-backend/pull/7#issuecomment-2075426755)

j-hariharan committed:
Added member-post relation
